app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,45 @@
+class SessionsController < ApplicationController
+
+  before_action :user, only: [:create]
+
+  def new
+    if current_user
+      redirect_to user_path(current_user), alert: I18n.
+        t("flash.actions.create.alert", resource_name: "Session")
```

Comment:
Style suggestion:
```
redirect_to(
  user_path(current_user),
  alert: I18n. t("flash.actions.create.alert", resource_name: "Session")
)
```

Also, since this is a special case, a suggestion to create a custom translation key in `en.yml`, which can still follow the same key structure. For example:
```
en:
  sessions:
    new:
      alert: "Already logged in"
```

And use that as:
```
redirect_to root_path, alert: t(".alert")
```

Reply:
Done in sha: df661d7